src/Attr.cc

```diff
@@ -195,6 +195,12 @@ Attributes::Attributes(std::vector<AttrPtr> a,
 
 void Attributes::AddAttr(AttrPtr attr)
 	{
+	// Display a warning for duplicated tags on a type. &log tags are intentionally
+	// ignored here because duplicate log tags on record types are valid, and don't
+	// cause any significant breakage for other types.
+	if ( attr->Tag() != ATTR_LOG && Find(attr->Tag()) )
+		reporter->Warning("Found duplicate tag %s", attr_name(attr->Tag()));
```

Comment:
```suggestion
		reporter->Error("Duplicate %s attribute is ambiguous", attr_name(attr->Tag()));
```

Think it's worth graduating to full-blown Error rather than Warning since that causes Zeek to exit before the run-loop and that's better since otherwise the user's intent is ambiguous and Zeek may end up doing something entirely unexpected.

Probably a further help to ignore the other attributes besides `&log` that also have unambiguous interpretation of duplicates: `&optional`, `&redef`, `&broker_allow_complex_type`, `&raw_output`, `&error_handler` (`&deprecated` without message strings could be, too)